curator-framework/src/main/java/org/apache/curator/framework/imps/OperationAndData.java

```diff
@@ -115,6 +115,11 @@ BackgroundCallback getCallback()
         return operation;
     }
 
+    void clearSleep()
+    {
+        sleepUntilTimeMs.set(0);
```

Comment:
I think this should be set to the time to current time (equivalent to `sleepFor(0)`), rather than blowing it away and setting to zero.

Reply:
But 0 is fine and very clear